app/controllers/Application.java

```diff
@@ -165,7 +181,23 @@ public static Result search() {
           .where()
           .idEq(appId).findUnique();
       return ok(searchPage.render(null, jobDetails.render(result)));
-    } else if (flowExecId != null && !flowExecId.isEmpty()) {
+    } else if (Utils.isSet(flowExecId)) {
+      String bestFlowExecId;
+      // check for exact match
+      bestFlowExecId = bestMatchForFlowExecIdLikeValue(flowExecId);
```

Comment:
What is the performance like for this search ? Since the partial seaches have to be full text search, our fear was that this will be prohibitively slow. also, if there are too many partial searches does it affect the overall db performance ?